prov/verbs/src/verbs_info.c

```diff
@@ -1186,13 +1186,15 @@ static int fi_ibv_fill_addr(struct rdma_addrinfo *rai, struct fi_info **info,
 }
 
 #define VERBS_NUM_DOMAIN_TYPES		3
+#define VERBS_MMAP_TEST_BUFFER_LEN_SIZE (1llu << 21)
```

Comment:
This is arbitrary.  There is an existing call, ofi_get_hugepage_size() that will return the correct size

Reply:
Resolved.